src/ResourceManager/DataFactories/Commands.DataFactoryV2/Changelog.md

```diff
@@ -19,5 +19,5 @@
 -->
 
 ## Current Release
-* Azure DataFactory Powershell cmdlets for ADF V2 Private Preview
```

Comment:
@hvermis looks like the change log didn't get updated from the previous release - this should look like the following:

```
## Current Release
* Added two new cmdlets: Update-AzureRmDataFactoryV2 and Stop-AzureRmDataFactoryV2PipelineRun

## Version 0.1.0
* Azure DataFactory PowerShell cmdlets for ADF V2 Private Preview
```

Reply:
Should it be Version 0.1.0 or Version 0.3.0? We released our V2 preview under 0.3.0.

Reply:
@hvermis it should be 0.3.0 in that case

Reply:
done